### ISSUE TITLE
two static analysis warnings in 4.10

### ISSUE THREAD

**Issue by sophiapoirier** [steinbergmedia/vstgui#211]
I updated to 4.10 and clang static analyzer produced two new warnings in a Mac build:

```
In file included from vstgui/vstgui/vstgui_mac.mm:5:
In file included from ../../vstgui/vstgui/vstgui.cpp:69:
In file included from ../../vstgui/vstgui/lib/platform/common/generictextedit.cpp:177:
../../vstgui/vstgui/lib/platform/common/stb_textedit.h:561:4: warning: Value stored to 'i' is never read
   i = 0;
   ^   ~
In file included from vstgui/vstgui/vstgui_mac.mm:25:
../../vstgui/vstgui/lib/platform/mac/cocoa/nsviewframe.mm:44:1: warning: 'DebugRedrawAnimDelegate' lacks a 'dealloc' instance method but must release '_layer'
@implementation DebugRedrawAnimDelegate
^
2 warnings generated.
```

(Also, I know I have filed issue like this before. Might I suggest static analysis as part of your CI? It is part of my release qualification and unfortunate that I haven't been able to get clean analysis using VSTGUI.)

**Comment by scheffle** [comment]
Hi, the second issue is already fixed. Regarding adding static analysis, we are open for pull requests. 
The first issue is in a 3rd party library I'm unsure if it is easily fixable.

**Comment by sophiapoirier** [comment]
Thanks and glad to hear that the second (and certainly more substantive) issue is already resolved.

Creating pull requests has never worked for me:

```ERROR: Permission to steinbergmedia/vstgui.git denied to sophiapoirier.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.```

However I see that there is a newer version 1.13 of `stb_textedit.h` which resolves the extraneous `i = 0;` assignment:
https://github.com/nothings/stb/blob/master/stb_textedit.h
so you could grab that version to use (and if I could open a PR, I would do as much).

But as for your CI, I'm not really sure how you have that stuff set up alas.

**Comment by scheffle** [comment]
Now fixed on develop or with the to be released version 4.10.1

**Comment by sophiapoirier** [comment]
thank you!
